dispatcher/MongoConnect.py

```diff
@@ -461,6 +469,7 @@ def ProcessCommands(self):
             except Exception as e:
                 dt = 10
                 self.log.error("DB down? %s" % e)
+                self.log.debug(f"Traceback {e}")
```

Comment:
This is the same as the line above

Reply:
thx, let me change the line above to type(e) since we don't want the entire trackback ending up in the logs if self.log.error magically starts having access to the database